documentation/modules/con-user-model.adoc

```diff
@@ -1,20 +1,15 @@
 // Module included in the following assemblies:
 //
-// <List assemblies here, each on a new line>
+// master.adoc
 
-// Base the file name and the ID on the module title. For example:
-// * file name: my-concept-module-a.adoc
-// * ID: [id='my-concept-module-a']
-// * Title: = My concept module A
-
-// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
 [id='con-user-model-{context}']
-// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
 = User model
 
-Users define authentication and authorization for messaging clients and console users. Users are
-configured as `MessagingUsers` resources. Users can only be created, deleted, read and listed (not
-modified). Following is an example user:
+You can create users to access messaging clients and the {ProductName} Console. The user definition files specify the authentication type and authorization policies.
```

Comment:
Aren't we supposed to avoid using 'can'?

Reply:
How about : 

Use the user definition files specify the authentication type and authorization policies for users created to access messaging clients and the {ProductName} Console.

Reply:
@ncbaratta: Using "can" is ok ("should" is the word we're not supposed to use)

Reply:
Got it. I still think we can consolidate the sentence :)

Reply:
Done